### DISCUSSION TITLE
Enable thumbnail

### DISCUSSION THREAD

**Discussion by Dau-Duc-Hai** [pimcore/pimcore#15974] [Q&A / Support] [answered]
I have configured Thumbnail in /settings/thumbnails in Dev mode. But when I switch to APP_DEBUG=0
PIMCORE_DEV_MODE=0 the setting disappears, and I cannot get thumbnails. Please help, 
Thank you!

**Comment by aryaantony92** [comment, marked as answer]
Hi @Dau-Duc-Hai,
You need to run the command 'pimcore:thumbnails:clear' or call Asset\Image\Thumbnail\Config::clearTempFiles() to revalidate the generated image thumbnails if you are using 'symfony-config' as the write target for the storage. 
You can also refer here for more information https://pimcore.com/docs/platform/Pimcore/Deployment/Configuration_Environments/#revalidate-existing-configuration-on-production

**Reply by Dau-Duc-Hai** [reply]
Thank you very much! it is OK now
